client/state/reader/lists/reducer.js

```diff
@@ -202,6 +203,28 @@ export function isRequestingLists( state = false, action ) {
 	return state;
 }
 
+/**
+ * Tracks if a list request has been made.
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @returns {Object}        Updated state
+ */
+export function listRequests( state = {}, action ) {
+	switch ( action.type ) {
+		case READER_LIST_REQUEST:
+			return {
+				...state,
+				[ `${ action.listOwner }:${ action.listSlug }` ]: true,
+			};
+		case READER_LIST_RECEIVE:
+			return {
+				...state,
+				[ `${ action.data.list.owner }:${ action.data.list.slug }` ]: false,
+			};
+	}
```

Comment:
Maybe the description on name could use an update.

At first glance reading "Tracks if a list request has been made", and then seeing "READER_LIST_RECIEVE" turn the item to `false` made me a little confused: "Why would we say there has been no request made once the request is finished?".  But I think the intention here is to track all the requests that have been made (existence in the state tree), and whether they are still in progress or not (the true vs false value), is that right?

Reply:
> But I think the intention here is to track all the requests that have been made (existence in the state tree), and whether they are still in progress or not (the true vs false value), is that right?

Yup! That's correct. I've updated the language to be the following: 

>  This object tracks all list requests that have been made and whether those requests are in progress or not.

Is that clearer?

Reply:
Looks good, thanks!